redfish-core/lib/oem/ibm/system_attention_indicator.hpp

```diff
@@ -44,8 +44,7 @@ inline void getSAI(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
                         const dbus::utility::MapperGetObject& object) {
         if (ec || object.empty())
         {
-            BMCWEB_LOG_ERROR << "DBUS response error " << ec.message();
-            messages::internalError(aResp->res);
+            BMCWEB_LOG_DEBUG << "Failed to get LED DBus name: " << ec.message();
```

Comment:
We are ignoring all errors here.. but I think that is fine, since isn't that important of a property and that is what we did for 1030 it looks like 
https://github.com/ibm-openbmc/bmcweb/blob/1030/redfish-core/lib/oem/ibm/system_attention_indicator.hpp#L38